src/test/java/org/opensearch/neuralsearch/util/ProcessorDocumentUtilsTests.java

```diff
@@ -80,4 +83,150 @@ public void test_with_different_configurations() throws URISyntaxException, IOEx
         }
     }
 
+    public void test_unflatten_simple_dot_notation() {
```

Comment:
Could you make the test naming consistent with others?
`testUnflatten_whenSimpleDotNotation_thenSucceed`

Reply:
Sure. Will do.